content/en/docs/instrumentation/swift/manual.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Manual Instrumentation
+linkTitle: Manual
+weight: 5
+description: >-
+<img width="35"
+src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Swift.svg"
+alt="Swift"></img>
+Manual instrumentation for opentelemetry-swift
+---
+
+## Setup
+
+The [OpenTelemetrySdk](https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/OpenTelemetrySdk/OpenTelemetrySdk.swift#L37) provides limited functionality in its default configuration. For more useful functionality, some configuration is required. 
+
+The default registered `TracerProvider` and `MetricProvider` are not configured with an exported. There are several [exporters](https://github.com/open-telemetry/opentelemetry-swift/tree/main/Sources/Exporters) available depending on your needs. Below we will explore configuring the OTLP exporter, which can be used for sending data to the [opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector).
```

Comment:
Eventually we want to have Exporters in their own page, but I think for a first version of the Swift documentation, this should be ok, @cartermp: what do you think?

Reply:
Yes, I agree. We can add more and more of this later.